app/Http/Controllers/ExpenseController.php

```diff
@@ -24,6 +25,48 @@ public function ministry()
     }
```

Comment:
Look at this. You have closed the class definition before defining your methods. Do you test your code locally at all?

Reply:
No sir... It's no closed.
Kindly check it again